src/Nest/XPack/Security/User/GetUserAccessToken/GetUserAccessTokenResponse.cs

```diff
@@ -19,5 +19,8 @@ public class GetUserAccessTokenResponse : ResponseBase
 
 		[DataMember(Name ="type")]
 		public string Type { get; set; }
+
+		[DataMember(Name = "authentication")]
+		public AuthenticateResponse Authentication { get; set; }
```

Comment:
While nice I am not sure this is the way to go, since this also inherits a lot from `ResponseBase` that makes no sense (e.g `.ApiCallDetails`, `.DebugInformation`).

Reply:
That's fair enough and I do agree it should probably be its own type for this purpose. Will apply that change.